NumberBaseball/NumberBaseball/main.swift

```diff
@@ -20,39 +61,35 @@ func generateThreeNonOverlappingRandomNumbers() -> Set<Int> {
     return randomNumbers
 }
 
-func countStrikeOrBall(answerNumber: [Int], tryCount: Int) -> (Int, Int) {
-    let suggestNumber = [Int](generateThreeNonOverlappingRandomNumbers())
+func countStrikeOrBall(computerNumbers: [Int], tryCount: Int, userSuggestNumbers: String) -> (Int, Int) {
+    var userSuggestThreeNumbers = [Int]()
+    let userSuggestNumbersSplit = userSuggestNumbers.components(separatedBy: " ")
+    userSuggestThreeNumbers = userSuggestNumbersSplit.map{Int($0) ?? 0}
```

Comment:
```suggestion
    let userSuggestNumbersSplit = userSuggestNumbers.components(separatedBy: " ")
    let userSuggestThreeNumbers = userSuggestNumbersSplit.map{Int($0) ?? 0}
```

1. 요게 낫지 않을까요??

2. 그리고, `userSuggestNumbersSplit` 보다는 `userSuggestSplitedNumbers` 나 `SplitedUserSuggestNumbers` 가 더 나을것 같네요!  변수이름은 명사형으로 하는게 좋을것 같아요

3. 벌써 고차함수를 쓰셨네요 ㅎㅎ 대단하십니다. 
- 질문) 사용하신 고차함수 map은 `정확히` 어떤 기능을 수행하나요?
- 제안) Swift의 클로져는 축약할수 있는거 알고 계시죠? 다만, 처음부터 축약형을 쓰기보단, 본 형태를 정확히 알고 쓰는게 좋다고 생각해요! 캠프초반에는 (특히 숫자게임과 묵찌빠 게임에는) 클로져를 전부 다 풀어쓰는게 어떨까요?

Reply:
- 고차함수 `map`은 컨테이너 내부에 기존 데이터를 변형하여 새로운 컨테이너를 생성하는 기능을 수행한다고 알고 있습니다  
- 클로져를 풀어서 다시 써보았습니다.